macros/macros.texlive

```diff
@@ -0,0 +1,21 @@
+#/*! \page config_macros Default configuration: /etc/debbuild/macros.texlive
+# \verbatim
+#
+# This is a per-system DEBBUILD configuration file.
+# Per-user configuration should be added to ~/.debmacros.
+#
+#==============================================================================
+# ---- TeX Live utilities
+#
+%__texlive_base %(kpsewhich --var-value=SELFAUTOPARENT)
+%__texlive_local %(kpsewhich --var-value=TEXMFLOCAL)
+#
+%__texlive_bin %{__texlive_base}/bin/%{_build_cpu}-%{_os}
```

Comment:
Does this work with system-provided TeXLive too?

Reply:
The first two, yes, but the last is `/usr/bin` on Kubuntu 18.04.3 LTS with
```
ii  texlive-base                                    2017.20180305-1                             all          TeX Live: Essential programs and files
ii  texlive-binaries                                2017.20170613.44572-8ubuntu0.1              amd64        Binaries for TeX Live

```
Will push solution independent of `rpmbuild`/`debbuild`.